prototypes/signal.lua

```diff
@@ -0,0 +1,125 @@
+local capture_tint = {0.9,0.9,0.9}
+local target_tint = {0.1,0.6,0.2}
+
+data:extend({
+    {
+        type = "item-subgroup",
+        name = "virtual-signal-TLBE",
```

Comment:
Lets use same format these signal prototypes as the other the prototypes: starting with `tlbe-` instead of ending with it. (keeps things clean and tidy)

Reply:
I'm happy with this suggestion